### PR TITLE
added defaultDeploymentRepo on RepositoryVirtual

### PR DESCRIPTION
**Pull request by PatriceG**
I added support to set the default deployment repo name, that we need.
It's fixing issue #460

### CODE REVIEW

File: dohq_artifactory/admin.py [hunk 1]
```diff
@@ -872,6 +872,7 @@ def __init__(
         self.notes = ""
         self.package_type = packageType or package_type
         self.repositories = repositories or []
+        default_deployment_repo_name=None
```
Comment (beliaev-maksim) [inline dohq_artifactory/admin.py:875]: it is not an attribute of the class

and should we have it as such?

**Review by beliaev-maksim — CHANGES_REQUESTED** [review]
also need tests for this

**Comment by PatriceG** [comment]
sorry indeed it should be 
self.default_deployment_repo_name=None

I'm not sure I understand how to setup a test environment for your lib.

**Comment by beliaev-maksim** [comment]
please provide a unittest with mocks, we use `responses`

**Comment by PatriceG** [comment]
I'm not familiar with mocking in Python.
I'm not sure I understand how to setup a test environment for your lib.
I just wanted to contribute this fix in the lib (I do have a workaround by sublcassing the RepositoryVirutal class but it's not ieal).

**Comment by beliaev-maksim** [comment]
these are the classic unit tests:
https://github.com/devopshq/artifactory/blob/master/docs/CONTRIBUTE.md#unit

then you can see the `unit` folder and add new tests similar to other

**Comment by PatriceG** [comment]
There is not a single existing unit test for RepositoryLocal and Virtual.
I'm sorry I have no time to try to understand how to setup a test env.
Feel free to reject my PR if you like. I can do with my local modifications. I just wanted to simply contribute.

**Pull request merged**